--- vpopmail-5.4.33/Makefile.am	2013-12-04 09:56:27.314000038 -0700
+++ vpopmail-5.4.33-patched/Makefile.am	2013-12-04 10:00:02.002992199 -0700
@@ -144,10 +144,10 @@
 	$(INSTALL) -d -m 0700 \
 	  $(DESTDIR)@vpopmaildir@/@domains_dir@
 
-	echo "-I@vpopmaildir@/include" > \
-	 $(DESTDIR)@vpopmaildir@/etc/inc_deps
-	echo "-L@vpopmaildir@/lib -lvpopmail @auth_libs@ @LIBS@" > \
-	 $(DESTDIR)@vpopmaildir@/etc/lib_deps
+	echo "-I/usr/include/libvpopmail" > \
+	 $(DESTDIR)/etc/libvpopmail/inc_deps
+	echo "-L/usr/lib/libvpopmail -lvpopmail @auth_libs@ @LIBS@" > \
+	 $(DESTDIR)/etc/libvpopmail/lib_deps
 
 	if test ! -r $(VUSAGECCONF); then \
 		$(INSTALL) -m 0644 \
