--- vpopmail-5.4.33/Makefile.am	2013-12-04 09:56:27.314000038 -0700
+++ vpopmail-5.4.33-patched/Makefile.am	2013-12-04 10:00:02.002992199 -0700
@@ -144,9 +144,9 @@
 	$(INSTALL) -d -m 0700 \
 	  $(DESTDIR)@vpopmaildir@/@domains_dir@
 
-	echo "-I@vpopmaildir@/include" > \
+	echo "-I/usr/include/vpopmail" > \
 	 $(DESTDIR)@vpopmaildir@/etc/inc_deps
-	echo "-L@vpopmaildir@/lib -lvpopmail @auth_libs@ @LIBS@" > \
+	echo "-L/usr/lib/vpopmail -lvpopmail @auth_libs@ @LIBS@" > \
 	 $(DESTDIR)@vpopmaildir@/etc/lib_deps
 
 	if test ! -r $(VUSAGECCONF); then \
--- vpopmail-5.4.33/vusaged/configure.ac	2011-02-28 10:00:45.000000000 -0700
+++ vpopmail-5.4.33-patched/vusaged/configure.ac	2013-12-13 15:09:24.610000563 -0700
@@ -379,8 +379,8 @@
 
 if test x"$with_vpopmail" != x""
 then
-   CPPFLAGS="$CFLAGS `cat $with_vpopmail/etc/inc_deps 2> /dev/null`"
-   LDFLAGS="$LDFLAGS `cat $with_vpopmail/etc/lib_deps 2> /dev/null`"
+   CPPFLAGS="$CFLAGS `cat $with_vpopmail/inc_deps 2> /dev/null`"
+   LDFLAGS="$LDFLAGS `cat $with_vpopmail/lib_deps 2> /dev/null`"
 fi
 
 AC_CHECK_HEADERS(vpopmail.h)

